src/Tactic/Cong.agda

```diff
@@ -35,7 +35,7 @@ open import Data.List.Base   as List  using ([]; _∷_)
 open import Data.Maybe.Base  as Maybe using (Maybe; just; nothing)
 open import Data.Nat.Base    as ℕ     using (ℕ; zero; suc; _≡ᵇ_; _+_)
 open import Data.Unit.Base            using (⊤)
-open import Data.Word.Base   as Word  using (toℕ)
+open import Data.Word64.Base   as Word64  using (toℕ)
 open import Data.Product.Base         using (_×_; map₁; _,_)
 open import Function                  using (flip; case_of_)
```

Comment:
Fix knock-on effect on the alignment here?

Reply:
I think we should give up on non alpha-stable alignment